src/Swarm/Language/Parse.hs

```diff
@@ -329,13 +329,13 @@ parseExpr = fixDefMissingSemis <$> makeExprParser parseTermAtom table
 exprLoc2 :: Parser (Syntax -> Syntax -> Term) -> Parser (Syntax -> Syntax -> Syntax)
```

Comment:
We should update the comment too.

Reply:
Done and I made it a local definition too :+1: